code/game/machinery/doors/airlock.dm

```diff
@@ -958,20 +958,20 @@
 							"<span class='hear'>You hear welding.</span>")
 			if(W.use_tool(src, user, 40, volume=50, extra_checks = CALLBACK(src, .proc/weld_checks, W, user)))
 				welded = !welded
-				user.visible_message("<span class='notice'>[user.name] has [welded? "welded shut":"unwelded"] [src].</span>", \
+				user.visible_message("<span class='notice'>[user.name] [welded? "welds shut":"unwelds"] [src].</span>", \
 									"<span class='notice'>You [welded ? "weld the airlock shut":"unweld the airlock"].</span>")
 				update_icon()
 		else
 			if(obj_integrity < max_integrity)
 				if(!W.tool_start_check(user, amount=0))
 					return
-				user.visible_message("<span class='notice'>[user] is welding the airlock.</span>", \
+				user.visible_message("<span class='notice'>[user] begins repairing the airlock.</span>", \
```

Comment:
You should not see the intent here in my opinion; it's ambigious because you see whats happening but it shouldnt straight up tell you that theyre repairing it.

Reply:
Sure thing, I'll change it. I was wondering why it was like that to begin with.

Reply:
cool, up to you if you wanna do this one as well but the "x finished repairing" lower in this file was always like this, but if you wanna change it to "x finishes welding" thatd be consistent too